chsdi/templates/htmlpopup/geothermische_potenzialstudien.mako

```diff
@@ -0,0 +1,84 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<%
+    lang = lang if lang in ('fr','it','en') else 'de'
+    titel = 'titel_%s' % lang
+    weblink = 'weblink_%s' % lang
+    download = c['attributes']['download']
+%>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.land')}</td>       <td>${c['attributes']['land'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.kanton')}</td>       <td>${c['attributes']['kanton'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.nb_studien')}</td>       <td>${c['attributes']['nb_studien'] or '-'}</td></tr>
+</%def>
+
+
+<%def name="extended_info(c,lang)">
+<%
+    lang = lang if lang in ('fr','it','en') else 'de'
+    titel = 'titel_%s' % lang
+    weblink = 'weblink_%s' % lang
+    download = c['attributes']['download']
+    nbstudien = c['attributes']['nb_studien']
+    titel = c['attributes'][titel].split('##')
+    autor = c['attributes']['autor'].split('##')
+    jahr = c['attributes']['jahr'].split('##')
+    auftraggeber = c['attributes']['auftraggeber'].split('##')
+    weblink = c['attributes'][weblink].split('##')
+    download = c['attributes']['download'].split('##')
+%>
+
+<table class="table-with-border bohrungen_tiefer_500-extended">
+     <tr><th class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.land')}</th>       <td>${c['attributes']['land'] or '-'}</td></tr>
+     <tr><th class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.kanton')}</th>       <td>${c['attributes']['kanton'] or '-'}</td></tr>
+     <tr><th class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.nb_studien')}</th>       <td>${c['attributes']['nb_studien'] or '-'}</td></tr>
```

Comment:
This returns "-" when the value in the table is "0". (see UR for example)
Should display "0" instead